src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ThemeServiceTests.cs

```diff
@@ -278,71 +278,307 @@ public async Task GetThemes()
+            var methodologyService = new Mock<IMethodologyService>(Strict);
+            var publishingService = new Mock<IPublishingService>(Strict);
+            var cacheService = new Mock<IBlobCacheService>(Strict);
+            var releasePublishingStatusRepository = new Mock<IReleasePublishingStatusRepository>(Strict);
+
+            await using (var contentContext = InMemoryApplicationDbContext(contextId))
+            await using (var statisticsContext = InMemoryStatisticsDbContext(contextId))
+            {
+                var service = SetupThemeService(
+                    contentContext,
+                    statisticsContext,
+                    releaseDataFileService: releaseDataFileService.Object,
+                    releaseFileService: releaseFileService.Object,
+                    releaseSubjectRepository: releaseSubjectRepository.Object,
+                    methodologyService: methodologyService.Object,
+                    publishingService: publishingService.Object,
+                    cacheService: cacheService.Object,
+                    releasePublishingStatusRepository: releasePublishingStatusRepository.Object);
+
+                releaseDataFileService
+                    .Setup(s => s.DeleteAll(releaseVersionId, true))
+                    .ReturnsAsync(Unit.Instance);
+
+                releaseFileService
+                    .Setup(s => s.DeleteAll(releaseVersionId, true))
+                    .ReturnsAsync(Unit.Instance);
+
+                releaseSubjectRepository
+                    .Setup(s => s.DeleteAllReleaseSubjects(releaseVersionId, false))
+                    .Returns(Task.CompletedTask);
+
+                methodologyService
+                    .Setup(s => s.DeleteMethodology(methodology.Id, true))
+                    .ReturnsAsync(Unit.Instance);
+
+                publishingService.Setup(s => s.TaxonomyChanged(CancellationToken.None))
+                    .ReturnsAsync(Unit.Instance);
+
+                cacheService
+                    .Setup(s =>
+                        s.DeleteCacheFolderAsync(
+                            ItIs.DeepEqualTo(new PrivateReleaseContentFolderCacheKey(releaseVersionId))))
+                    .Returns(Task.CompletedTask);
+
+                releasePublishingStatusRepository.Setup(mock =>
+                        mock.RemovePublisherReleaseStatuses(new List<Guid> { releaseVersion.Id }))
+                    .Returns(Task.CompletedTask);
+
+                var result = await service.DeleteTheme(theme.Id);
+                VerifyAllMocks(releaseDataFileService,
+                    releaseFileService,
+                    releaseSubjectRepository,
+                    methodologyService,
+                    publishingService,
+                    cacheService,
+                    releasePublishingStatusRepository);
+
+                result.AssertRight();
+
+                Assert.Equal(0, contentContext.Publications.Count());
```

Comment:
I actually prefer `Empty` in this particular case because I think it's easier to read...in fact I'm sure my IDE usually suggests the change, but didn't here for some reason 🤷‍♂️.
![image](https://github.com/user-attachments/assets/748723a5-6f24-406f-94ff-78404782b238)

Reply:
After looking at your screenshot, it is much easier to read at a glance. My issue is many Assert libraries have all sorts of keywords for all sorts of situations and it adds a whole bunch of other stuff to memorise. Although you could argue that autocomplete helps with that. 

Still think I'd prefer assert libraries to have a few methods for checking, but there probably is a balance. I imagine assert library writers are hit by the same temptation as language designers in that there is always more they could add, and the more they add, in my opinion, it often starts to become ridiculous.

Reply:
Looks like this hasn't been changed either - I imagine you've not pushed your changes.